### PR TITLE
Add non-security, non-license use cases to SBOM section

### LINKED ISSUE #310: Add Non-Security, Non-License Use Cases to SBOM Material
@syotfs and I have considered the idea of adding non-security, non-license SBOM use cases to the Academy. One use case I've been interested in relates to organizations using SBOMs to identify open source software projects that are important to their products and using that knowledge to guide donations and donation size to a set of open source software projects. eBay is currently pursuing SBOMs to achieve this objective among others. I'm sure @syotfs has some other ideas too.

**Describe the solution you'd like**
A sub-section in an area related to SBOMs (we are flexible on where exactly) describe these non-security, non-license use cases.

**Describe alternatives you've considered**
Not adding anything beyond security and license use cases.

**Additional context**
This is just a proposal. We defer to @sheesh since her recent SBOM work has been excellent and we see this merely as a small complement to what she has done. If @sheesh and @ltagliaferri are amenable, Michael and I are glad to put in an initial PR.

### PR DESCRIPTION
**Pull request by jspeed-meyers**
Fix #310 

Feel free to discuss or throw tomatoes. Also, I deleted some blank spaces in nearby lines to clean things up. Tell me to knock it off if those blank spaces were intentional. My OCD kicked in.

cc @syotfs in case he wants to edit or add something

### What are the acceptance criteria? 
@sheesh should be happy with the the change.

### How should this PR be tested?
Check that the change successfully deploys.

### CODE REVIEW

**Comment by jspeed-meyers** [comment]
Also, I did this editing in the GitHub UI, so the Enforce commit signing is unhappy. If you want me to try to fix this, please let me know. Additionally, it is possible to allow the GitHub UI GPG key in the Enforce policy. Let me know if you want an issue or PR to fix this.

File: content/open-source/sbom/what-is-an-sbom.md [hunk 1]
```diff
@@ -21,11 +21,13 @@ A software bill of materials, or an SBOM (pronounced s-bomb), is a key resource
...
+* **Licensing**: Software projects often pull from tens or hundreds of dependencies whose different licenses may have different restrictions on the use of the code. Being able to track the licenses of your dependencies allows you to stay informed about legal restrictions and licensing compatibility issues, and help you decide which dependencies work best for your project needs.
+* **Automate alerts**: The machine readable format of SBOMs enables you to create automated alerts to inform you of important events in your codebase, such as security vulnerabilities, missing components, or incompatible dependency licensing.
+* **Make open source funding decisions**: By understanding which open source software components are used and how frequently across an organization's codebases, an organization can make more informed decisions about providing funding to open source projects.
```
Comment (jspeed-meyers) [inline content/open-source/sbom/what-is-an-sbom.md:29]: I added this line.

File: content/open-source/sbom/what-is-an-sbom.md [hunk 2]
```diff
@@ -21,11 +21,13 @@ A software bill of materials, or an SBOM (pronounced s-bomb), is a key resource
...
+* **Find abandoned dependencies**: Assuming that an SBOM's components contain sufficient information (such as [package URLs](https://github.com/package-url/purl-spec)) to link packages with external data, an SBOM consumer could discover if any of the components are open source projects that are abandoned or archived. The user could then take actions such as forking the abandoned project or removing that project as a dependency.
```
Comment (jspeed-meyers) [inline content/open-source/sbom/what-is-an-sbom.md:30]: I added this line.

File: content/open-source/sbom/what-is-an-sbom.md (hunk 1 above)
Comment (jspeed-meyers) [reply content/open-source/sbom/what-is-an-sbom.md:29]: Line 29

File: content/open-source/sbom/what-is-an-sbom.md (hunk 2 above)
Comment (jspeed-meyers) [reply content/open-source/sbom/what-is-an-sbom.md:30]: Line 30

**Comment by sheesh** [comment]
This looks great to me -- thanks for the clean up and additions! Let me know if I should merge, or wait for @syotfs.

**Comment by jspeed-meyers** [comment]
Cool! Give @syotfs 24 hours. If no word by then, ship it!

**Pull request merged**